response_operations_ui/controllers/message_controllers.py

```diff
@@ -11,6 +11,25 @@
 logger = wrap_logger(logging.getLogger(__name__))
 
 
+def get_conversation(thread_id):
+    logger.debug("Retrieving conversation")
+
+    url = f'{current_app.config["BACKSTAGE_API_URL"]}/v1/secure-message/threads/{thread_id}'
+
+    response = requests.get(url, headers={'Authorization': _get_jwt()})
+
+    try:
+        response.raise_for_status()
+    except HTTPError:
+        logger.exception("Conversation retrieval failed")
+        raise ApiError(response)
+
+    logger.debug("Retrieval successful")
+    conversation = response.json()
+
+    return conversation
```

Comment:
Don't assign response.json to conversation as it's never used apart from in the return statement.